### PR TITLE
[Jsontracer] Add detail to show bar's information.

### PR DESCRIPTION
**Pull request by parkjisu6239**
This will show you the details when you click on the bar. Mark detail differently when they are selected single or multiple.

ONE-vscode-DCO-1.0-Signed-off-by: Jisu Park <parkjisu6239@gmail.com>

### CODE REVIEW

**Comment by parkjisu6239** [comment]
from draft #126
from issue #94

**Review by wnsdud4197 — DISMISSED** [review]

File: media/Jsontracer/detail.js [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+function refinedSelectedOp() {
+  const selectedOpList = document.querySelectorAll('.selected-op');
+  const refinedOP = [];
+  const refinedOPDict = {};
+  let idx = 0;
+
+  selectedOpList.forEach(element => {
+    const name = element.dataset.name;
+    const ts = element.dataset.ts * 1;
```
Comment (llFreetimell) [inline media/Jsontracer/detail.js:140]: Why multiply 1?

Comment (parkjisu6239) [reply media/Jsontracer/detail.js:140]: This is to change the string to num. Automatic transformation was used.

Comment (parkjisu6239) [reply media/Jsontracer/detail.js:140]: I'll change it to `Number` for readability.

**Review by wnsdud4197 — APPROVED** [review]

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**